Bootstrapper.cs

```diff
@@ -0,0 +1,42 @@
+namespace FringeNancy
+{
+    using Nancy;
+    using Nancy.TinyIoc;
+    using Microsoft.Framework.Runtime;
+
+    public class Bootstrapper : DefaultNancyBootstrapper
+    {
+        private IApplicationEnvironment appEnv;
+        
+        protected override IRootPathProvider RootPathProvider { get { return new RootPathProvider(appEnv); }}
+        
+        public Bootstrapper(IApplicationEnvironment appEnv)
+        {
+            this.appEnv = appEnv;
+        }
+        
+        protected override void ApplicationStartup(TinyIoCContainer container, Nancy.Bootstrapper.IPipelines pipelines)
+        {
+            this.Conventions.ViewLocationConventions.Add((viewName, model, context) =>
+            {
+                return string.Concat("wwwroot/", viewName);
```

Comment:
I think it should be, maybe we should make a template for vnext + nancy somewhere, so that mat's project doesn't get spammed, and we can improve the yo generator too?